userspace/libsinsp/plugin.h

```diff
@@ -161,6 +162,7 @@ class sinsp_plugin {
 	sinsp_thread_pool::routine_id_t subscribe_routine(ss_plugin_routine_fn_t routine_fn,
 	                                                  ss_plugin_routine_state_t* routine_state);
 	bool unsubscribe_routine(sinsp_thread_pool::routine_id_t routine_id);
+	bool dump(sinsp_dumper& dumper);
```

Comment:
```suggestion
	bool dump_state(sinsp_dumper& dumper);
```
Suggesting a more scoped name here too -- your call